timers.js

```diff
@@ -1,4 +1,35 @@
+$(document).ready(function() {
+  var seconds = 0;
+  var timerId;
+  var timeOn = false;
 
+  $("#start").on("click", function() {
+    console.log("clicked start");
+    if (timeOn == false) {
+      timeOn = true;
+      timerId = setInterval(function() {
+        seconds += 1;
+        $("#timer").text(seconds);
+      }, 1000);
+    }
+  });
 
+  $("#pause").on("click", function() {
+      if (timeOn == true) {
+      timeOn = false;
+      console.log("clicked pause");
+      clearInterval(timerId);
+    }
+  });
 
-
+  $("#reset").on("click", function() {
+        timeOn = false;
+        console.log("clicked reset");
+        clearInterval(timerId);
+        $("#timer").text("Stop Watch")
+        seconds = 0;
+      }
+    );
+  });
+//do the above functions have names? can I call the functions somehow? If not, can I add names to them?
+//is setInterval like a for loop? or is that a bad analogy?
```

Comment:
We just talked about this IRL, but I would consider them separate things. A for loop has a defined end and usually doesn't function based on some timed interval.